src/app/components/MediaLoader/index.test.tsx

```diff
@@ -168,4 +169,25 @@ describe('MediaLoader', () => {
       });
     });
   });
+
+  describe('Config', () => {
+    it('should use the counterNameOverride when rendering On Demand TV', async () => {
```

Comment:
Do we need an opposite / positive test for this, whereby it uses the counter name from the RequestContext if no override is passed in?